content/blog/2019-07-30-SciPy20109.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "QCArchive at SciPy 2019!"
+date: 2019-07-31
+description: "QCArchive's experience at SciPy2019"
+pageDescription : "Experience gained and interesting projects at SciPy2019."
+type : "post"
+image : "images/blog/scipy.jpg"
+author : "Daniel G. A. Smith"
+authorImage : "images/people/dgashead.jpg"
+authorDescription : "Daniel is a MolSSI Software Scientist and the lead QCArchive developer."
+authorSocial : 
+ - icon : "tf-ion-social-twitter"
+   URL : "https://twitter.com/dga_smith"
+ - icon : "tf-ion-social-github"
+   URL : "https://github.com/dgasmith"
+---
+
+Part of the QCArchive team went to SciPy this year to find out about the latest
+scientific and data analysis python projects. SciPy brings together hundreds of
+members of the scientific python community from academia, government, and
+industry partners; you can find more details
+**[here](https://www.scipy2019.scipy.org)**. Attending SciPy for us is a
+fantastic opportunity to interact with leaders in the field and to find out
+where the SciPy ecosystem is evolving over the next year.
+
+First and foremost, we gave a talk with MolSSI Associate Lori Burns, a research
+software engineering at Georgia Tech, who is working with us on the QCArchive
+project.  This talk covers how QCArchive is making the quantum chemistry
+ecosystem more composable by breaking down common operations into building
+blocks and exploring ways to interact with the SciPy ecosystem. We also briefly
+covered some of the capabilities of the QCArchive ecosystem and projects that
+use this ecosystem.
+
+<center>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/HBkY_qnYSQw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</center>
+
+-------------
+
+It was fantastic to see friends that are part of many academic groups,
```

Comment:
Is it possible to link out to all of these groups? Should we?

Reply:
I attempted to at the beginning, but eventually decided against it.